### PR TITLE
Add "View the Continua" button configurable using CBL.config.php

### PR DESCRIPTION
**Pull request by jmealo**
https://www.wrike.com/open.htm?id=33373727

### CODE REVIEW

File: html-templates/cbl/student-dashboard.tpl [hunk 1]
```diff
@@ -58,6 +58,9 @@
                 </select>
             </div>
             <button class="button primary">Show Rubric Details</button>
+            {if Slate\CBL\CBL::$continuaUrl}
+                <a class="button primary" href="{Slate\CBL\CBL::$continuaUrl}" target="_blank">View the Continua</a>
```
Comment (themightychris) [inline html-templates/cbl/student-dashboard.tpl]: Should be `{Slate\CBL\CBL::$continuaUrl|escape}`

File: html-templates/cbl/teacher-dashboard.tpl [hunk 2]
```diff
@@ -12,6 +12,9 @@
     <header class="page-header">
         <div class="header-buttons">
             <button type="button" class="primary" data-action="demonstration-create">Log a Demonstration</button>
+            {if Slate\CBL\CBL::$continuaUrl}
+                <a class="button primary" href="{Slate\CBL\CBL::$continuaUrl}" target="_blank">View the Continua</a>
```
Comment (themightychris) [inline html-templates/cbl/teacher-dashboard.tpl]: Should be `{Slate\CBL\CBL::$continuaUrl|escape}`

**Comment by jmealo** [comment]
@themightychris issues addressed, good catch.

**Comment by themightychris** [comment]
crap this one targeted master

**Comment by themightychris** [comment]
hang on to your hats I'm gonna rewrite history

**Comment by themightychris** [comment]
I rewound master and merged these commits again into development via #16

**Pull request merged**